libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1411,6 +1412,16 @@ bool AP_Arming::opendroneid_checks(bool display_failure)
     return true;
 }
 
+//Check for multiple RC in serial protocols
+bool AP_Arming::serial_protocol_checks(bool display_failure)
+{
+    if (AP::serialmanager().have_serial(AP_SerialManager::SerialProtocol_RCIN, 1)) {
+       check_failed(display_failure, "Multiple UARTs configured for RC input");
```

Comment:
```suggestion
       check_failed(display_failure, "Multiple serial ports configured for RC input");
```
Serial better aligns with the param names for serial functions .